index.html

```diff
@@ -10,129 +10,129 @@
 </head>
 
 <body>
```

Comment:
- [ ] Although your overall design is great, Your body seems to be overflowing. let's address this to make your design shine 😎

![your-body](https://user-images.githubusercontent.com/69077061/201469637-f9280192-ee51-4972-8ff6-7d70f3e38c74.PNG)